src/lib/eliom_service_sigs.shared.mli

```diff
@@ -283,27 +277,26 @@ module type S = sig
       Eliom_parameter.non_localized_params ->
     service:
       ('a, 'b, 'meth, 'attach, 'co, 'ext, 'g,
-       'd, 'e, 'f, 'return) service ->
+       'd, 'e, 'f, 'return) t ->
     ('a, 'b * 'p, 'meth, 'attach, 'co, 'ext, 'g,
-     'd, 'e, 'f * 'pn, 'return) service
+     'd, 'e, 'f * 'pn, 'return) t
 
   (**/**)
 
   val which_meth :
-    (_, _, 'm, _, _, _, _, _, _, _, _) service -> 'm Meth.which
+    (_, _, 'm, _, _, _, _, _, _, _, _) t -> 'm Meth.which
 
   val get_info :
-    (_, _, _, 'att, _, _, _, _, _, _, _) service ->
-    'att attached_info
+    (_, _, _, 'att, _, _, _, _, _, _, _) t -> 'att attached_info
 
-  val is_external : (_, _, _, _, _, _, _, _, _, _, _) service -> bool
+  val is_external : (_, _, _, _, _, _, _, _, _, _, _) t -> bool
 
   val get_get_params_type_ :
```

Comment:
And the `_` suffix. Haven't bothered yet because these are exclusively for internal use.